gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx

```diff
@@ -20,6 +22,21 @@ export const MapMenu = (props: MapMenuProps) => {
   const {markers, menuItem} = useContext(MapContext);
   const markerTypes = Array.from(new Set(markers.map(marker => marker.markerType)));
 
+  const [currentIdx, setCurrentIdx] = useState(-1);
```

Comment:
-1 is a magic number?
From the name, it is not clear what index is meant

Reply:
Create const defaultIndex and change naming to currentIndex